barcalendar.py

```diff
@@ -440,8 +440,8 @@ def write(self):
 redis_releases = [
     ('5.6', 2020, 4, 2021, 10),
     ('6.0', 2020, 5, 2022, 5),
-    ('6.2', 2021, 8, 2023, 2),
-    ('7.0', 2022, 4, 2023, 10),
+    ('6.2', 2021, 8, 2024, 4),
+    ('7.0', 2022, 4, 2024, 10),
```

Comment:
Where did these dates come from? I don't see them on either of the URLs above?

Reply:
They haven't provided the EOL date anywhere, we just calculated it from the above links. As per [this link](https://redis.io/docs/about/releases/#support), they support the latest version and two previous versions get maintenance and security patches and they release a new version every year. 7.2 isn't yet released, but two release candidates are released https://github.com/redis/redis/releases. So until 8.0 released next year, 6.2 will get security and maintenance patches.